packages/manager/modules/pci/src/projects/new/payment/controller.js

```diff
@@ -361,6 +364,152 @@ export default class PciProjectNewPaymentCtrl {
+              return 'desktop';
+            }
+            if (
+              ua.match(
+                /Bot|Crawler|Spider|Yahoo|ia_archiver|Covario-IDS|findlinks|DataparkSearch|larbin|Mediapartners-Google|NG-Search|Snappy|Teoma|Jeeves|TinEye/i,
+              ) &&
+              !ua.match(/Mobile/i)
+            ) {
+              return 'desktop';
+            }
+            return 'mobile';
+          },
+        };
+      }
+
+      if (typeof window.tC.msr !== 'object') {
+        window.tC.msr = [];
+      }
+      window.tC.msr.dns = 'ovh.commander1.com';
+      window.tC.msr.id_site = '3810';
+      window.tC.msr.idorder =
+        Date.now().toString(36) +
+        Math.random()
+          .toString(36)
+          .substring(2, 12)
+          .padStart(12, 0);
+      window.tC.msr.amount = '{{order_amount_tax_free}}';
+      window.tC.msr.currency = '{{currency code}}';
+      window.tC.msr.user_id = '{{customer ID}}';
+      window.tC.msr.additional_params = `&dev=${window.tC.detect_device()}`;
+      window.tC.msr.additional_params +=
+        '&cty=' +
+        '{{FR/GB/IE/CZ/DE/ES/IT/LT/NL/PL/PT/FI/SN/TN/MA/CA/US/MX/AU/ID/SG/IN/Unknown}}';
+      window.tC.msr.additional_params +=
+        '&moyens_de_paiement=' +
+        '{{payment method}}' +
+        '&discount_price=' +
+        '{{discount amount tax free}}';
+      window.tC.msr.additional_params +=
+        '&n_customer=' +
+        '{{true if the client is a new customer OR false if not}}' +
+        '&nc=' +
+        '{{true if the client is a new customer OR false if not}}' +
+        '&site_domain=' +
+        'www.ovh.com';
+      window.tC.msr.additional_params += '&pcat=publiccloud';
+      window.tC.msr.additional_params += '&ot=pci_project_creation';
+      window.tC.msr.additional_params += '&newcustom=0';
+      const conversionTimestamp = new Date();
+
+      const conversionTimestampString = `${conversionTimestamp.getFullYear()}-${String(
+        conversionTimestamp.getMonth() + 1,
+      ).padStart(2, '0')}-${String(conversionTimestamp.getDate()).padStart(
+        2,
+        '0',
+      )} ${String(conversionTimestamp.getHours()).padStart(2, '0')}:${String(
+        conversionTimestamp.getMinutes(),
+      ).padStart(2, '0')}:${String(conversionTimestamp.getSeconds()).padStart(
+        2,
+        '0',
+      )} Europe/Paris`;
+
+      window.tC.msr.additional_params += `&conversion_timestamp=${conversionTimestampString}`;
+
+      window.tC.msr.tc_rand = Math.random();
+      window.tC.msr.pixel = document.createElement('img');
+      window.tC.msr.pixel.style.display = 'none';
+      window.tC.msr.pixel.src = `https://${window.tC.msr.dns}/mix/o3/?tcs=${window.tC.msr.id_site}&idorder=${window.tC.msr.idorder}&amount=${window.tC.msr.amount}&currency=${window.tC.msr.currency}&user_id=${window.tC.msr.user_id}${window.tC.msr.additional_params}&rand=${window.tC.msr.tc_rand}`;
+      document.head.appendChild(window.tC.msr.pixel);
+    }
```

Comment:
**nitpick:** Can you extract the logic to load mix commander into a different file?